blog/2020-10-08-FacilityLocationProblemQuickstart.adoc

```diff
@@ -0,0 +1,98 @@
+= New quickstart for the facility location problem
+:imagesdir: 2020/10/08
+:page-interpolate: true
+:awestruct-author: yurloc
+:awestruct-layout: blogPostBase
+:awestruct-tags: [use case, facility location problem, quarkus]
+
+:awestruct-share_image_filename: CreditDisputeProcessDiagram.png
+
+We have created a new https://github.com/kiegroup/optaplanner-quickstarts/tree/development/quarkus-facility-location[quickstart] that extends OptaPlanner use case portfolio with the https://en.wikipedia.org/wiki/Facility_location_problem[facility location problem] and also demonstrates building a cloud-native application using https://quarkus.io/[Quarkus], the new Java Native stack.
```

Comment:
nitpick: I 'd avoid the word "extends" as is implies that you inherited another pom or class.

Reply:
Being a Java keyword doesn't disqualify the word from usage in natural language.

I can think of several synonyms like "expand", "broaden" or "add to (the portfolio)" and I'm OK to use one of these instead if it doesn't compromise the accuracy. @emmurphy1 native speaker's opinion?

Reply:
@ge0ffrey Maybe I should point to https://www.optaplanner.org/compatibility/quarkus.html instead of quarkus.io to keep the reader on our site and, more importantly, to give him more OptaPlanner-tailored context about Quarkus role. Wdyt?

Reply:
> Being a Java keyword doesn't disqualify the word from usage in natural language.
> 
> I can think of several synonyms like "expand", "broaden" or "add to (the portfolio)" and I'm OK to use one of these instead if it doesn't compromise the accuracy.

Good argument and +1 to sidestep the discussion by using an synonym. For inspiration: https://www.powerthesaurus.org/extends/synonyms

Reply:
I like "widen".